Tasks/Npm/npm.ts

```diff
@@ -30,6 +33,58 @@ async function main(): Promise<void> {
     }
 }
 
+async function _logNpmStartupVariables() {
+    try {
+        // Log the NPM version
+        let version: string;
+        try {
+            let syncResult: IExecSyncResult = tl.execSync('npm', '--version');
+            if (syncResult.stdout) {
+                version = syncResult.stdout.trim();
+            }
+        } catch (err) {
+            tl.debug(`Unable to get NPM config info. Err:( ${err} )`);
+        }
+
+        // Log the NPM registries
+        let command = tl.getInput(NpmTaskInput.Command);
+        let npmRegistriesAry: INpmRegistry[];
+        let registryUrlAry = [];
+        switch (command) {
+            case NpmCommand.Install:
+            case NpmCommand.Custom:
+                npmRegistriesAry = await npmCustom.getCustomRegistries();
+                break;
+            case NpmCommand.Publish:
+                npmRegistriesAry = [await npmPublish.getPublishRegistry()];
+                break;
+        }
+        for (let registry of npmRegistriesAry) {
+            registryUrlAry.push(registry.url)
+        }
+
+
+        let npmTelem = {
+            'command': command,
```

Comment:
Is there any easy way to iterate through the NpmTaskInputs and add them to the object?

Reply:
Yeah, I wanted to do that but I couldn't identify them all from the ENV dictionary.  There's no prefix that I'm aware of.